src/Transaction.java

```diff
@@ -0,0 +1,87 @@
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+public class Transaction {
+
+    private String id;
+    private BigDecimal amount;
+    private String type;
+    private LocalDateTime date;
```

Comment:
```suggestion
    private LocalDateTime dateTime;
```
Или еще лучше отразить смысл в названии, например, `executedAt`